src/rf/apps/workers/process.py

```diff
@@ -6,13 +6,12 @@
 
 import math
 import time
-from datetime import datetime
 
 from apps.core.models import LayerImage, Layer
-from apps.workers.image_validator import ensure_band_count
+from apps.workers.image_validator import ImageValidator
 from apps.workers.sqs_manager import SQSManager
 import apps.core.enums as enums
-
+import apps.workers.status_updates as update
```

Comment:
Consider using the same alias as the module name so there's less things to juggle mentally.